Kickstarter-iOS/Views/Controllers/PledgeAmountViewController.swift

```diff
@@ -21,11 +21,7 @@ final class PledgeAmountViewController: UIViewController {
   private lazy var amountInputView: AmountInputView = { AmountInputView(frame: .zero) }()
   private lazy var titleLabel: UILabel = { UILabel(frame: .zero) }()
   private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
-  private lazy var spacer: UIView = {
-    UIView(frame: .zero)
-      |> \.translatesAutoresizingMaskIntoConstraints .~ false
-  }()
-
+  private lazy var horizontalSpacer: UIView = { UIView(frame: .zero) }()
```

Comment:
No real need for auto layout anymore...we're using spacing (where we have previously used auto layout)...this should be simpler and more readable.